superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -165,6 +173,36 @@ const displayField = ({
     helpText="Pick a nickname for this database to display as in Superset."
   />
 );
+const forceSSLField = ({
+  isEditMode,
+  changeMethods,
+  db,
+  sslForced,
+}: FieldPropTypes) => (
+  <div css={(theme: SupersetTheme) => infoTooltip(theme)}>
+    <Tooltip title={t('SSL is required for Preset')}>
```

Comment:
Is the plan to replace this before merging the branch into master?

Reply:
wait, I am confused, replace what portion?

Reply:
We can't say "SSL is required for Preset" in OSS Superset.

Reply:
oh wait, the text was actually updated on this one.

Reply:
Can we go with SSL is required here